aptos-move/framework/aptos-framework/sources/block.spec.move

```diff
@@ -30,6 +26,9 @@ spec aptos_framework::block {
         include staking_config::StakingRewardsConfigRequirement;
 
         aborts_if false;
+
+        // Only a valid proposer or the vm is authorized to produce blocks
+        ensures proposer == @vm_reserved;
```

Comment:
It is not necessary to have this ensures because proposer is an input variable and it is not changed during execution. This property is expressed using the aborts_if condition.

Reply:
it has been remove